samples/datatypes.js

```diff
@@ -611,6 +607,97 @@ async function queryWithTimestamp(instanceId, databaseId, projectId) {
   // [END spanner_query_with_timestamp_parameter]
 }
 
+async function addNumericColumn(instanceId, databaseId, projectId) {
+  // [START spanner_add_numeric_column]
+  // Imports the Google Cloud client library.
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
```

Comment:
So typically, we don't bother passing `projectId` in like this, and rather rely on the library figuring it out on it's own.  There are places we need it, but I kind of think we should keep the constructor parameterless where we can.

Reply:
Oh interesting. Yeah I suppose we have the GCLOUD_PROJECT env var that could be read. I understood this as, customers may not know how to set that (or maybe they're not comfortable using env vars) so we explicitly set them.

But if it is against the style guide, we can do a repo-wide cleanup of this.

I'll leave it as it is for now, just for consistency with how existing samples are set up.